docs/book/operator-guide/upload-relay.md

```diff
@@ -0,0 +1,196 @@
+  the moment the only value, is `RS2` .
+
+## Receiving the certificate
+
+Once the relay is done storing the blob, it will collect the confirmation certificate from the
+storage nodes.
+
+Then, it will send a response to the client, containing the `blob_id` of the stored blob, along with
+the `confirmation_certificate`. The client can then use this certificate to certify the blob on
+chain.
+
+## Installation
+
+### Download the Binary
+
+If you'd like to download a pre-built binary in order manually run `walrus-upload-relay`, you'll
+need to download it from [the releases page](https://github.com/MystenLabs/walrus/releases). Note
+that `walrus-upload-relay` does not daemonize itself, and requires some supervisor process to ensure
+boot at startup, to restart in the event of failures, etc.
+
+### Docker
+
+The docker image for `walrus-upload-relay` is available on Docker Hub as
+`mysten/walrus-upload-relay`.
+
+```sh
+docker run -it --rm mysten/walrus-upload-relay --help
+```
+
+### Build from Source
+
+Of course, if you'd like to build from sources, that is always an option, as well. The sources for
+the `walrus-upload-relay` are available on [GitHub](https://github.com/MystenLabs/walrus) in the
+`crates/walrus-upload-relay` subdirectory. Running the following from the root of the Walrus repo
+should get you a working binary.
+
+```sh
+cargo build --release --bin walrus-upload-relay
+./target/release/walrus-upload-relay --help
+```
+
+## Configuration
+
+Notice that `walrus-upload-relay` requires some configuration to get started. Below is an example of
+how you might place the configuration such that it is reachable when invoking Docker to run the
+relay. For the sake of the example below, we're assuming that:
+
+- `$HOME/.config/walrus/walrus_upload_relay_config.yaml` exists on the host machine and contains the
+  configuration for the `walrus-upload-relay`, as described [in the configuration
+  section](#relay-specific-configuration).
+- `$HOME/.config/walrus/client_config.yaml` exists on the host machine and contains Walrus client
+  configuration as specified [in the CLI setup section](../usage/setup.html#configuration).
+- Port 3000 is available for the relay to bind to (change this to whichever port you'd like to
+  expose from your host.)
+
+```sh
+docker run \
+  -p 3000:3000 \
+  -v $HOME/.config/walrus/walrus_upload_relay_config.yaml:/opt/walrus/walrus_upload_relay_config.yaml \
+  -v $HOME/.config/walrus/client_config.yaml:/opt/walrus/client_config.yaml \
+  mysten/walrus-upload-relay \
+    --context testnet \
+    --walrus-config /opt/walrus/client_config.yaml \
+    --server-address 0.0.0.0:3000 \
+    --relay-config /opt/walrus/walrus_upload_relay_config.yaml
+```
+
+### Relay-specific configuration
+
+An example `walrus-upload-relay` configuration file can be found
+[in the codebase](https://github.com/mystenlabs/walrus/tree/main/crates/walrus-upload-relay/walrus_upload_relay_config_example.yaml).
```

Comment:
Hint: Instead of or in addition to pointing to GitHub, you could also include the tip configuration on the docs site. For example, create a symlink in `docs/book/setup` pointing to `crates/walrus-upload-relay/walrus_upload_relay_config_example.yaml`:

```sh
cd docs/book/setup
ln -s ../../../crates/walrus-upload-relay/walrus_upload_relay_config_example.yaml ./walrus_upload_relay_config_example.yaml
```

Then you can include it here as follows:

````md
```yaml
{{ #include ../setup/walrus_upload_relay_config_example.yaml }}
```
````